include/aws/io/host_resolver.h

```diff
@@ -76,10 +81,21 @@ struct aws_host_listener;
 
 struct aws_host_listener_options;
 
+struct aws_host_resolver_purge_host_options {
+    struct aws_host_resolver *resolver;
```

Comment:
I don't know, but I prefer to have the `resolver` as a separate parameter instead included in the options. feel more natural and we mostly follow that pattern.

Reply:
Thanks, updated.